app/database_etl/etl_main.py

```diff
@@ -256,6 +243,29 @@ def validate_records(airtable_source):
 
     exit("EXITING – No acceptable records found.")
 
+def get_coords(place_name, place_type):
+    url = f"https://api.mapbox.com/geocoding/v5/mapbox.places/{place_name}.json?" \
+          f"access_token={os.getenv('MAPBOX_API_KEY')}"
+    r = requests.get(url)
+    data = r.json()
+    coords = None
+    if data is not None and "features" in data and len(data['features']) > 0:
+        # Check if we have any results of the right place type
+        # Otherwise fallback to the first result we get
+        for feature in data['features']:
+            if place_type in feature['place_type']:
+                coords = feature['center']
+                break
+        if coords is None:
+            coords = data['features'][0]['center']
+    return coords
+
+def add_latlng_to_df(place_type, name_col, df):
```

Comment:
maybe we can change the parameter "name_col" to "place_type_name" so it's a bit clearer that it relates to the place